data/data/azure/cluster/master/variables.tf

```diff
@@ -16,6 +16,18 @@ variable "vm_size" {
   type = string
 }
 
+variable "disk_encryption_set_id" {
+  default     = null
```

Comment:
Note: Patrick had some feedback about default terraform values in the original PR https://github.com/openshift/installer/pull/5134#pullrequestreview-862702928

I'm not sure that I understand what needs to be done. Do you @jhixson74? If not - let's ask Patrick to clarify.

Reply:
Fixed.